autopilot/contractor.go

```diff
@@ -1082,7 +1109,7 @@ func (c *contractor) candidateHosts(ctx context.Context, w Worker, hosts []hostd
 	return selectedHosts, selectedScores, nil
 }
 
-func (c *contractor) renewContract(ctx context.Context, w Worker, ci contractInfo, budget *types.Currency, renterAddress types.Address) (cm api.ContractMetadata, proceed bool, err error) {
+func (c *contractor) renewContract(ctx context.Context, w Worker, ci contractInfo, budget *types.Currency, currentHeight uint64) (cm api.ContractMetadata, proceed bool, err error) {
```

Comment:
`renewContract` and `refreshContract` could probably also just fetch the height on demand. Because really why not? It's a cheap enough call to perform once per contract.

Reply:
ok